src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style !== '');
```

Comment:
You can include this check inside the `reduce` call, to reduce the number of iterations

Reply:
Thank you!
Changed my solution a little, hope it is better now :)